src/hooks/usePncWebSocket.ts

```diff
@@ -0,0 +1,81 @@
+import { useEffect } from 'react';
+
+import * as pncWebSocketService from 'services/pncWebSocketService';
+
+/**
+ * HOOK CHECKERS
+ */
+
+/**
+ * Additional filtering parameters
+ */
+interface IOnParameters {
+  buildConfigId?: string;
+  userId?: string;
+}
+
+/**
+ * Check whether Build status change WebSocket event was sent.
+ *
+ * @param wsData - WebSocket data
+ * @param parameters - See {@link IOnParameters}
+ * @returns true when Build status changed, otherwise false
+ */
+export const onBuildStatusChanged = (wsData: any, { buildConfigId, userId }: IOnParameters = {}) => {
```

Comment:
`wsData` is any

Reply:
More advanced types are not available yet, but we will automatize them in the future.